Amazon.QLDB.Driver.IntegrationTests/utils/IntegrationTestBase.cs

```diff
@@ -101,6 +101,35 @@ public void RunCreateLedger()
                 .Build();
         }
 
+        public AsyncQldbDriver CreateAsyncDriver(
+            AmazonQLDBSessionConfig amazonQldbSessionConfig,
+            int maxConcurrentTransactions = default,
+            string ledgerName = default,
+            int retryLimit = -1)
```

Comment:
`retryLimit` seems not being used in this method anyway

Reply:
It will be used once the async session tests file is created